.github/workflows/CI.yml

```diff
@@ -0,0 +1,23 @@
+name: CI Job
+
+on:
+  push:
+    branches: [ master, dev ]
+  pull_request:
+    branches: [ master, dev ]
+
```

Comment:
Because the CI only build the artifact now, I guess we just need to apply it on dev branch in this time?

Reply:
It'll run on pull requests against either of those branches, I can make it run on all branches